alibi/exceptions.py

```diff
@@ -14,14 +14,28 @@ def __init__(self, message: str) -> None:
         super().__init__(message)
 
 
-class AlibiPredictorCallException(AlibiException):
+# Legacy exception classes starting with the prefix `Alibi`.
+# These have been kept around for backwards compatibility.
+# Any new exception classes should not start with the `Alibi` prefix.
+class AlibiPredictorCallException:
+    pass
+
+
+class AlibiPredictorReturnTypeError:
+    pass
+
+
+# Exception classes. These should only inherit from `AlibiException`. Ones inheriting from
+# other classes beginning with the prefix `Alibi` are to do with backwards compatibility as
+# exception classes used to all start with the prefix `Alibi`.`
+class PredictorCallError(AlibiException, AlibiPredictorCallException):
```

Comment:
I've also standardised to end the exception with `Error` rather than `Exception` which is common practice for specific Python exceptions.